src/block_cache/block_chunkcache.c

```diff
@@ -507,6 +507,72 @@ __config_get_sorted_pinned_objects(WT_SESSION_IMPL *session, const char *cfg[],
     return (0);
 }
 
+/*
+ * __chunkcache_insert --
+ *     Insert chunk into the chunkcache.
+ */
+static int
+__chunkcache_insert(WT_SESSION_IMPL *session, wt_off_t offset, wt_off_t size,
+  WT_CHUNKCACHE_HASHID *hash_id, uint64_t bucket_id, WT_CHUNKCACHE_CHUNK **new_chunk)
+{
+    WT_CHUNKCACHE *chunkcache;
+
+    chunkcache = &S2C(session)->chunkcache;
+
+    /*
+     * !!! (Don't format the comment.)
+     * Caller function should take a bucket lock before inserting the chunk.
+     */
+    WT_RET(__chunkcache_alloc_chunk(session, offset, (wt_off_t)size, hash_id, new_chunk));
```

Comment:
don't think we need this wt_off_t cast, as its passed in as wt_off_t.

Reply:
Nice catch, updated in [af44def](https://github.com/wiredtiger/wiredtiger/pull/9677/commits/af44def3208cc8d59e29c45b05509a738ab79f1d)